crates/router/src/connector/square/transformers.rs

```diff
@@ -140,10 +138,9 @@ impl TryFrom<(&types::TokenizationRouterData, WalletData)> for SquareTokenReques
             | WalletData::WeChatPayRedirect(_)
             | WalletData::WeChatPayQr(_)
             | WalletData::CashappQr(_)
-            | WalletData::SwishQr(_) => Err(errors::ConnectorError::NotSupported {
-                message: format!("{:?}", item.request.payment_method_data),
-                connector: "Square",
-            })?,
+            | WalletData::SwishQr(_) => Err(errors::ConnectorError::NotImplemented(
+                utils::get_unimplemented_payment_method_error_message("Square"),
+            ))?,
```

Comment:
We can club all the `match arms` and propagate the expected common error.
Please use `|` and throw the `NotImplemented` error in the end. No need for it to be done twice